test/boost/user_types_test.cc

```diff
@@ -111,16 +111,18 @@ SEASTAR_TEST_CASE(test_invalid_user_type_statements) {
 
         // non-frozen UDTs can't be inside collections, in create table statements...
         REQUIRE_INVALID(e, "create table bad (a int primary key, b list<ut1>)",
-                "Non-frozen user types or collections are not allowed inside collections: list<ut1>");
+                "Non-frozen user types or collections are not allowed inside collections: list<ks.ut1>");
         REQUIRE_INVALID(e, "create table bad (a int primary key, b set<ut1>)",
-                "Non-frozen user types or collections are not allowed inside collections: set<ut1>");
+                "Non-frozen user types or collections are not allowed inside collections: set<ks.ut1>");
         REQUIRE_INVALID(e, "create table bad (a int primary key, b map<int, ut1>)",
-                "Non-frozen user types or collections are not allowed inside collections: map<int, ut1>");
+                "Non-frozen user types or collections are not allowed inside collections: map<int, ks.ut1>");
         REQUIRE_INVALID(e, "create table bad (a int primary key, b map<ut1, int>)",
-                "Non-frozen user types or collections are not allowed inside collections: map<ut1, int>");
+                "Non-frozen user types or collections are not allowed inside collections: map<ks.ut1, int>");
         // ... and in user type definitions
         REQUIRE_INVALID(e, "create type ut2 (a int, b list<ut1>)",
-                "Non-frozen user types or collections are not allowed inside collections: list<ut1>");
+                "Non-frozen user types or collections are not allowed inside collections: list<ks.ut1>");
+        REQUIRE_INVALID(e, "create type ut3 (a int, b list<ut2>)",
```

Comment:
This test does not match the comments:
```cpp
// non-frozen UDTs can't be inside collections, in create table statements...
(...)
// ... and in user type definitions
```

This test probably deserves to be moved later below, under the comment
```cpp
// can't reference non-existing UDT
```
which already contains a similar test (except that it doesn't try to put the UDT inside a collection).